README.md

```diff
@@ -152,16 +153,19 @@ They require JetPack 4.2 and above, and [@dusty-nv](https://github.com/dusty-nv)
 
 ### From Source
 
-If you are installing from source, you will need Python 3.7 or later and a C++14 compiler. Also, we highly recommend installing an [Anaconda](https://www.anaconda.com/distribution/#download-section) environment.
-You will get a high-quality BLAS library (MKL) and you get controlled dependency versions regardless of your Linux distro.
+#### Prerequisites
+If you are installing from source, you will need:
+- Python 3.7 or later (for Linux, Python 3.7.6+ or 3.8.1+ is needed)
```

Comment:
Err, why? 3.7.0 should work, shouldn't it?

Reply:
I didn't test this, but from the current Readme.md in master:

<img width="878" alt="image" src="https://user-images.githubusercontent.com/4468967/171663303-67152a45-d889-4514-9b90-aa863652fddb.png">

Reply:
Note that the above section only applies to Linux environments

Reply:
This sounds super out of date, conda have long since fixed this particular compiler issue. May be rephrase it to something like (latest patch revision of Python runtime is recommended)

Reply:
Nice. Are you sure that conda has fixed the compiler issue that required python 3.7.6+?

If so, we can remove that error section and python version requirements completely